tests/sources/xml/test_ead.py

```diff
@@ -501,12 +563,11 @@ def test_ead_record_invalid_date_and_date_range_are_omitted(caplog):
             assert date.range.lte != "1989"
             assert date.range.gte != "2001"
             assert date.range.lte != "1999"
-
     assert ("has a date that couldn't be parsed: 'undated'") in caplog.text
     assert ("has a later start date than end date: '2001', '1999'") in caplog.text
 
 
-def test_ead_record_correct_identifiers_from_multiple_unitid(caplog):
+def test_ead_transform_with_multiple_unitid_gets_valid_ids():
```

Comment:
`Ead` has more of these edge case tests than the other sources we've done so far, we might consider redoing these with stub records but not sure if that's worth the effort

Reply:
@ehanson8 For this PR, I'll stick to just renaming the test and will revisit when I work on the `get_identifiers` field method.

Reply:
That's totally fine, just food for thought!